bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/bubble/Bubble.java

```diff
@@ -311,62 +334,66 @@ public final void setY(final double yPos) {
    * Getter method: for the xSpeed value.
    * @return xSpeed value
    */
-  public final double getXSpeed() {
+  public double getXSpeed() {
     return this.xSpeed;
   }
+
   /**
    * Setter method: for the xSpeed value.
-   * @param xSPEED  double value to set xSpeed to
+   * @param xSpeed  double value to set xSpeed to
```

Comment:
Wait, I got a checkstyle error because it was xSPEED. I changed it to xSpeed and I no longer have the checkstyle error